common.yml

```diff
@@ -1075,7 +1075,7 @@ pre:
           export NOTARY_KEY_NAME='server-4.4'
           export NOTARY_TOKEN='${signing_auth_token_server_4_4}'
           export BARQUE_USERNAME='${evg_user}'
```

Comment:
```suggestion
```
since this isn't used anymore, looks like we can delete it

Reply:
The username is actually still required because it verifies that the API key matches the user.